bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -165,6 +166,13 @@ private constructor(
     @get:JvmName("adminRetention")
     val adminRetention: AdminRetentionMethods by lazy { AdminRetentionMethods(this) }
 
+    /**
+     * Access API methods under the "admin/domain_allows" endpoint.
+     */
+    @Suppress("unused") // public API
+    @get:JvmName("adminDomainAllow")
+    val adminDomainAllow: AdminDomainAllowMethods by lazy { AdminDomainAllowMethods(this) }
```

Comment:
Could you please move this up a bit so that we have alphabetical order for the different `val`s representing all the methods?